src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
+export const Good = ({ good, setSelectedGood, selectedGood, setNoGoods }) => (
+  <tr
+    data-cy="Good"
+    className={good === selectedGood ? 'has-background-success-light' : ''}
+  >
+    <td>
       <button
-        data-cy="ClearButton"
+        onClick={() => {
+          setSelectedGood(good);
+          setNoGoods(false);
+        }}
+        data-cy={good === selectedGood ? `RemoveButton` : 'AddButton'}
         type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        className={`button ${good === selectedGood ? 'is-info' : ''}`}
+      >
+        {good === selectedGood ? '-' : '+'}
+      </button>
+    </td>
+    <td data-cy="GoodTitle" className="is-vcentered">
+      {good}
+    </td>
+  </tr>
+);
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [noGoods, setNoGoods] = useState(false);
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {noGoods === true && (
+      <h1 className="title is-flexis-align-items-center">No goods selected</h1>
```

Comment:
A typo in the class name, 'flexis' is not a valid property. It should be 'flex'.